src/config.cpp

```diff
@@ -351,6 +351,7 @@ video_t video {
     -1,
   }, // vt
 
+  {},  // capture method
```

Comment:
Since this is just "capture" in the header, it probably makes sense to be the same here.

Reply:
ok I'll change it